tests/eleventy-site/_redirects

```diff
@@ -1,4 +1,4 @@
-/force.html         /test.html                      200!
-/something          /otherthing.html
+/force              /test                           200!
```

Comment:
`eleventy' puts files in their own directories (e.g. `test.html` is copied to `test/index.html`)